lib/membrane_mp4/muxer/cmaf.ex

```diff
@@ -309,23 +311,35 @@ defmodule Membrane.MP4.Muxer.CMAF do
     if are_all_group_pads_ready?(pad, ctx, state) do
       stream_format = generate_output_stream_format(output_pad, state)
 
+      old_input_pads_ready? = state.all_input_pads_ready?
+
+      state = update_input_pads_ready(pad, ctx, state)
+
+      {actions, state} =
+        if old_input_pads_ready? != state.all_input_pads_ready? do
```

Comment:
isn't it enough to check if `state.all_input_pads_ready?`?

Reply:
Well, what we care is the fact that it changed from `false` to `true`, and since `update_input_pads_ready` always sets it to true (event if it already was) I just check if it changed.